.gitignore

```diff
@@ -1 +1,2 @@
 node_modules/
+.idea/
```

Comment:
Super-pedantic stuff here, but consider configuring your editor to add newlines at EOF.